app/webpack/components/authentication/sign_in.vue

```diff
@@ -6,7 +6,20 @@
       </div>
     </div>
 
-    <div v-if="!loading && !showTermsStep" ref="firebaseAuthContainer"></div>
+    <div
+      v-if="!loading && !showTermsStep && !showVerificationStep"
+      ref="firebaseAuthContainer"
+    >
+      <template v-if="!profileSignin">
```

Comment:
I _think_ you can use `inlineFlow` for this instead of a new var? (The original intention of `inlineFlow` is that it's for use cases like the profile sign in, but I didn't want to name it explicitly as that as it's more about the behaviour of the component, than tied to a separate feature of the site).